toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -146,7 +146,7 @@ func main() {
 
 	// Setup remote source configuration
 	templateSrcConfig.sourceURL = *sourceURL
-	templateSrcConfig.caCerts = x509.NewCertPool()
+	templateSrcConfig.caCerts, err = x509.SystemCertPool()
```

Comment:
Thanks for recording the error! Please also add a check for it now, so we don't continue if this call fails.

Reply:
Sure, I've updated this